src/lib/components/home/artwork_rails/artwork_rail.tsx

```diff
@@ -203,29 +209,32 @@ export class ArtworkRail extends React.Component<Props & RelayPropsWorkaround, S
     }
   }
 
+  railStyle() {
+    const sideMargin = isPad ? 40 : 20
+    const style: any = { marginLeft: sideMargin, marginRight: sideMargin }
+
+    if (!this.hasStartedFetching()) {
+      style.minHeight = minRailHeight
+    }
+
+    return style
+  }
+
   render() {
     if (this.state.loadFailed) {
       return null
     }
-    const hasStartedFetching = this.props.relay && this.props.relay.variables.fetchContent
-
-    const sideMargin = isPad ? 40 : 20
-    const style: any = { marginLeft: sideMargin, marginRight: sideMargin }
-    if (!hasStartedFetching) {
-      // if there's no content, set minHeight to prevent spinners from all the rails showing
-      style.minHeight = 400
-    }
 
-    const hasArtworks = hasStartedFetching && this.props.rail.results && this.props.rail.results.length
-    if (!hasArtworks) {
+    const hasArtworks = this.props.rail.results && this.props.rail.results.length
+    if (this.hasStartedFetching() && !hasArtworks) {
```

Comment:
Similar to above, this should read ‘when _finished_ fetching and there are no artworks’.